examples/pcr/include.am

```diff
@@ -0,0 +1,14 @@
+# vim:ft=automake
+# All paths should be given relative to the root
+
+if BUILD_EXAMPLES
+noinst_PROGRAMS += examples/pcr/quote
+noinst_HEADERS  += examples/pcr/quote.h
+examples_pcr_quote_SOURCES      = examples/pcr/quote.c \
+									       examples/tpm_io.c
```

Comment:
Check tabs vs. spaces.

Reply:
Thanks, this is inherited from the signed_timestamp example, which inherited it from the native_tests example.
I will fix all of this in a single commit independently from the functionality, but will put it in this PR.